src/App.js

```diff
@@ -16,6 +17,7 @@ function App() {
         <Route path="/" element={<Landing />} />
         <Route path="/login" element={<Login />} />
         <Route path="/signup" element={<Signup />} />
+        <Route path="/matchmaking" element={<Matchmaking />} />
```

Comment:
Wrap the element in a Protected component, so like below:
element={<Protected><Matchmaking /></Protected>